src/lightning_lite/utilities/exceptions.py

```diff
@@ -0,0 +1,17 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+class MisconfigurationException(Exception):
+    """Exception used to inform users of misuse with Lightning."""
```

Comment:
Why move this over if we want to remove it? cc @justusschock

Reply:
Because I don't want to update it in 1000 places and then have one reviewer complain, and then having to undo it xD. These countless mini refactors can be worked on after Lite and there shouldn't be any more work than before. So I'm not at all concerned about this.